azure-devops/azext_devops/dev/artifacts/universal.py

```diff
@@ -27,12 +27,18 @@ def publish_package(feed, name, version, path, description=None, organization=No
     :type path: str
     """
     colorama.init()   # Needed for humanfriendly spinner to display correctly
-    organization = resolve_instance(detect=detect, organization=organization)
+
+    organization, project = resolve_instance_and_project(
+        detect=detect,
+        organization=organization,
+        project=project,
```

Comment:
This will automatically pick up the project from default config or local git repo context and target call will always be made at project level even if he user did not intend to make it that way.

What is the difference in targeting a feed with or without the project name? 

Is there a use case that a user will want to not input the project name? If yes, this command will now not allow the user to do it since it will pick up the project from auto detect logic.